src/carbon_sendgrid_adapter.cr

```diff
@@ -35,7 +36,6 @@ class Carbon::SendGridAdapter < Carbon::Adapter
     end
 
     # :nodoc:
-    # Used only for testing
```

Comment:
I'm not sure why this said used only for testing :thinking: This method is how we send the email lol